src/modsim2/attack/transfer.py

```diff
@@ -82,6 +83,7 @@ def compute_transfer_attack(
         attack_names: Output strings for the attack names
         batch_size: Batch size for the dataloader used in predicting outputs
         loss_function: Loss function to use in computing mean_loss_rate
+        **kwargs: Keyword arguments passed to pytorch_lightning.Trainer()
```

Comment:
suggest rename to `trainer_kwargs` or explicitly name them as separate params if we know which ones these will be and we know we'll always use them

Reply:
Have renamed to `trainer_kwargs` to maintain flexibility in the trainer config for #26 (and any other future changes)